package.json

```diff
@@ -68,7 +73,8 @@
     "publish-package": "./build_helpers/publishPackage.sh",
     "test": "mocha-webpack --webpack-config webpack.config-test.js \"test/**/*-test.js\" --require build_helpers/test-globals.js",
     "test:watch": "mocha-webpack --webpack-config webpack.config-test.js --watch \"test/**/*-test.js\" --require build_helpers/test-globals.js",
-    "test:server": "webpack-dev-server --config webpack.config-test.js --hot --inline"
+    "test:server": "webpack-dev-server --config webpack.config-test.js --hot --inline",
+    "lint:eslint": "eslint src/* test/*"
```

Comment:
can we prefer just "lint" instead of "lint:eslint"

Reply:
@KamranAsif Yes of course.
I looked at `test:watch`, `test:server` etc. So in case somebody would add e.g. a style linter this could be named `lint:stylelint`.
But you're right, in this case there should be a generic task `lint` which includes all sub lint tasks.